src/main/java/duke/Tasks/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package duke.Tasks;
+import duke.additionalParsers.DateParser;
+
+
```

Comment:
Should this have a JavaDoc header comment? I noticed that not all classes have header comments - not sure if it's a requirement, but you might want to take a look into this.